src/features/island/hud/components/inventory/InventoryItems.tsx

```diff
@@ -36,61 +30,29 @@ export const InventoryItems: React.FC<Props> = ({
   onPlace,
   isSaving,
 }) => {
-  const [currentTab, setCurrentTab] = useState<Tab>("basket");
-
-  const handleTabClick = (tab: Tab) => {
-    setCurrentTab(tab);
-  };
+  const [currentTab, setCurrentTab] = useState<number>(0);
```

Comment:
Here also, I would prefer to keep the naming of the tab vs numbers that don't really convey any meaning.

Reply:
The new CloseButtonPanel component uses a numerical index to keep track of the tabs, so I am using a number to set the currentTab.  The CloseButtonPanel is a generic component so it doesn't make sense to set a customized string for tab names for different panels